source/includes/fact-atlas-roles.rst

```diff
@@ -1,9 +1,10 @@
 The user specified in the connection string must have the
 :atlasrole:`atlasAdmin` role.
 
-To use ``mongosync`` in the reverse direction, you must `create a
-custom role </atlas/reference/api/custom-roles-create-a-role/>`__ that
-grants the following ActionTypes:
+To use ``mongosync`` in the :ref:`reverse direction <c2c-api-reverse>`,
+you must `create a custom role
+</atlas/reference/api/custom-roles-create-a-role/>`__ that grants the
+following ActionTypes:
```

Comment:
[nit]

We refer to these as "ActionTypes" here and "user actions" in the following paragraph. Consider using the same term throughout.

Reply:
Done